tests/unit/amo/components/TestAddonAdminLinks.js

```diff
@@ -22,6 +22,7 @@ import {
 
 describe(__filename, () => {
   let store;
+  const id = 123;
```

Comment:
Please rename this variable to `addonId` if you intend to use it in test cases below.